modules/fortify-server/fortify-user-data.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+fortify_db_name
```

Comment:
The numbered parameters should be removed as the templating will handle replacing the variable values.